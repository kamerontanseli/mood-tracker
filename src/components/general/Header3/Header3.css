.Header3 {
  font-weight: 300;
  color: #333;
}