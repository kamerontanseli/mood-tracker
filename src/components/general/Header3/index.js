import "./Header3.css";
import React from "react";

const Header3 = props => <h3 className="Header3" {...props} />

export default Header3;
