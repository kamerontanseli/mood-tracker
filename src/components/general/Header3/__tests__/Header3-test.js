import React from 'react'
import renderer from 'react-test-renderer'
import Header3 from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Header3>Hello world</Header3>
    );
    expect(snapshot).toMatchSnapshot()
  })
})