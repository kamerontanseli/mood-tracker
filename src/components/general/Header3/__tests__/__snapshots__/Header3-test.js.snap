// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<h3
  className="Header3"
>
  Hello world
</h3>
`;
